app/src/main/java/de/westnordost/streetcomplete/quests/parking_fee/AddBikeParkingFee.kt

```diff
@@ -9,7 +9,7 @@ class AddBikeParkingFee : OsmFilterQuestType<FeeAnswer>() {
     // element selection logic by @DerDings in #2507
     override val elementFilter = """
         nodes, ways, relations with amenity = bicycle_parking
-        and access !~ no|private
+        and access and access !~ no|private
```

Comment:
Maybe 
```suggestion
        and access ~ yes|customers|public
```
to be consistent with the car parking access quest?

Reply:
For example you probably wouldn't want to ask for `access=permissive`